src/test/java/pages/Employment_Status.java

```diff
@@ -0,0 +1,164 @@
+    private String ESSaveData = "//button[text()=' Save ']";
+    private String ESCancelData = "//button[text()=' Cancel ']";
+    private String ESAlredyExisting = "//div[@class='oxd-form-row']/div";
+    private String ESTable = ".oxd-table-body";
+    private String ESEdit = "(//button[@class='oxd-icon-button oxd-table-cell-action-space'])[2]";
+    //private String ESEdit= "(//div[contains(text(),'Marketing')]//following::div[1]//child::button[2]";
+    private String ESEditStatus1 = "(//input[contains(@class,'oxd-input oxd-input')])[2]";
+    //private String ESEditStatus2 = "//input[@class='oxd-input oxd-input--active oxd-input--error']";
+    private String ESDelete = "(//button[@class='oxd-icon-button oxd-table-cell-action-space'])[1]";
+    //private String ESDelete = "(//div[contains(text(),'Marketing')]//following::div[1]//child::button[1]";
+    private String ESDeleteYesDelete = "//button[text()=' Yes, Delete ']";
+    private String ESDeleteNoCancel = "//button[text()=' No, Cancel ']";
+    private String ESMultiDelete1 = "(//span[@class='oxd-checkbox-input oxd-checkbox-input--active --label-right oxd-checkbox-input'])[2]";
+    private String ESMultiDelete2 = "(//span[@class='oxd-checkbox-input oxd-checkbox-input--active --label-right oxd-checkbox-input'])[3]";
+    private String ESDeleteSelected = "//button[text()=' Delete Selected ']";
+    private String ESRecord = "//span[@class='oxd-text oxd-text--span']";
+    private String ESSave = "//p[text()='Success']";
+    private final String TrashButton = "//i[contains(@class,'oxd-icon bi-trash')]";
+    private final String EditButton= "//i[contains(@class,'oxd-icon bi-pencil-fill')]";
+    private final String ES = "//div[contains(@style,'flex-basis: 80%;')]";
+    private final String employmentStatus = "//div[@class='oxd-table-body']";
+    @FindBy(xpath = employmentStatus)
+    private List<WebElement> ESlist;
+   /* @FindBy(xpath = Rows)
+    private List<WebElement> ESRows;
+    @FindBy(xpath = Columns)
+    private List<WebElement> ESColumns;
+
+    */
+
+    public Employment_Status(WebDriver driver) {
+        super(driver);
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    public boolean saveNewES(String eStatus) {
+        click(By.xpath(ESAdd));
+        setText(By.xpath(ESAddStatus), eStatus);
+        if (getText(By.xpath(ESAlredyExisting)).contains("Already exists")) {
+            click(By.xpath(ESCancelData));
+        } else {
+            click(By.xpath(ESSaveData));
+
+            isElementVisible(By.cssSelector(ESTable));
+
+            for (WebElement EmpStatus : ESlist) {
+                String txtEmpStatus = EmpStatus.getText();
+                System.out.println(txtEmpStatus);
+
+            }
+            Boolean match = ESlist.stream().map(s -> s.getText()).anyMatch(s -> s.equalsIgnoreCase(eStatus));
+
+
+
+        }
+        return true;
```

Comment:
This method is always returning true.

Reply:
i was performing assertion inside the test